docs/contributing.rst

```diff
@@ -69,9 +69,9 @@ and instead, ``git rebase`` your branch.
 Implementing New Features
 -------------------------
 
-If you are writing a new feature, please ensure you write appropriate Pytest test cases and place them under ``tests/``.
+If you are writing a new feature, please ensure you write appropriate Pytest test-cases and place them under ``tests/``.
 
-If you are making a larger change, please consult first with the `Ethereum Python Dev Discord <https://discord.gg/abJEARdx3Q>`_ ``#vyper`` channel.
+If you are making a larger change, please consult first with the `Ethereum Python Dev Discord <https://discord.gg/abJEARdx3Q>`_ via ``#vyper`` channel.
```

Comment:
we should probably update this to the dedicated discord - https://discord.gg/6tw7PTM7C2 / #compiler-dev channel

Reply:
Okay, I will update it!